estate_data/admin.py

```
@@ -1,3 +1,19 @@
 from django.contrib import admin
+from .models import EstateImage, Estate
+from django.utils.html import format_html
 
 # Register your models here.
+@admin.register(EstateImage)
```

Flake8 (7.2.0)

[error] expected 2 blank lines, found 1

(E302)